tests/testament/categories.nim

```diff
@@ -418,8 +418,9 @@ proc `&?.`(a, b: string): string =
 
 proc processSingleTest(r: var TResults, cat: Category, options, test: string) =
   let test = "tests" & DirSep &.? cat.string / test
+  let target = if cat.string.normalize == "js": targetJS else: targetC
```

Comment:
But the target should be extracted from the test "spec" section.

Reply:
So the correct fix would be to change all the JS tests to specify `target: "js"`? Because currently none of them do, and it's instead handled in [`proc jsTests`](https://github.com/nim-lang/Nim/blob/0f5261e9711c3fd57241874963bd5e45b11ed65e/tests/testament/categories.nim#L219).

Reply:
ouch. well your fix is ok then